src/components/header/Header.tsx

```diff
@@ -1,4 +1,7 @@
+import i18next from 'i18next';
```

Comment:
if these dependencies are not used anymore, you can remove them :)

Reply:
done !